### ISSUE TITLE
Tutorials are quite outdated

### ISSUE THREAD

**Issue by aleighbrown** [BioJulia/BioTutorials#3]
The notebooks as they stand are from Julia 0.5

**Comment by kescobo** [comment]
Yeah, and we really need more of them. Someone suggested making something like the BioStars handbook in julia, and we started https://github.com/BioJulia/biojulia_handbook. There are some additional suggestions for what should be included and some[ discussion here](https://github.com/BioJulia/biojulia_handbook/issues/1). I think this repo probably makes more sense for that stuff.
